java/arcs/sdk/wasm/WasmCollectionImpl.kt

```diff
@@ -76,5 +87,8 @@ class WasmCollectionImpl<T : WasmEntity>(
     override fun clear() {
         entities.clear()
         WasmRuntimeClient.collectionClear(particle, this)
+        onUpdateActions.forEach { action ->
+            action(entities.values.toSet())
```

Comment:
Here too

Reply:
Actually, there's other cases of this too. Maybe put into its own method? `notifyOnUpdateActions()`?